docs/tsg.md

```diff
@@ -110,15 +110,16 @@ ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
 ```
 
 <a name="random_404_returned_for_client_requests"></a>
-## Random 404 returned for client requests
+## 404 returned for client requests
 
 For a SignalR persistent connection, it first `/negotiate` to Azure SignalR service and then establish the real connection to Azure SignalR service. Our load balancer must ensure that the `/negotiate` request and the following connect request goes to the similar instance of the Service otherwise 404 occurs. Our load balancer now relies on the *signature* part of the generated `access_token` to keep the session sticky.
 
 ### Troubleshooting Guide
+1. Check if the client `SkipNegotiation` is `true`. When using Azure SignalR, client receives redirect url when it first negotiates with the app server. Client should NOT skip negotiation when using Azure SignalR.
 1. Following [How to view outgoing requests](#view_request) to get the request from client the the service.
```

Comment:
This should the first step?

Reply:
There is no need to see outgoing request if `SkipNegotiation` is `true`?

Reply:
But if you know how to view outgoing requests you will be able to get a clear evidence about negotiation is skipped (by looking at the url)?